nengo/spa/tests/test_module.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+
+import nengo
+from nengo import spa
+from nengo.spa.utils import similarity
+
+
+class SpaCommunicationChannel(spa.Module):
```

Comment:
I'm tempted to ask for documentation for this class, so that a person could understand it's typical use without looking through all the tests it's used in, but I can't decide if that's excessive or not. Thoughts?